locales/fr/LC_MESSAGES/base.po

```diff
@@ -22,87 +22,140 @@ msgstr ""
 #: bot.py:58
 msgid "ECO_DESCRIPTION"
 msgstr ""
+"Eco n'est pas une banque. Eco est un solde unique qui vous donne jusqu'à 5% de APY sur vos dépôts et 5% de cashback chez un nombre croissant de commerçants."
```

Comment:
I think 'single balance' would better translate to 'un compte unique'

Reply:
proposal by `getordone` : 
`chez un nombre croissant de commerçants` -> `chez un nombre grandissant de commerçants`